lighthouse-core/audits/accessibility/list.js

```diff
@@ -22,6 +22,8 @@ class List extends AxeAudit {
       name: 'list',
       description: 'Lists contain only `<li>` elements and script supporting elements ' +
           '(`<script>` and `<template>`).',
+      fallbackDescription: 'Lists do not contain only `<li>` elements and script ' +
+          'supporting elements (`<script>` and `<template>`).',
```

Comment:
Right. I'd hate to lose the full description when someone fails. That's when they need the most help :) Most of the time, users will start off failing the audit. I think it will also be quite confusing if the two descriptions change a lot. People are going to be like "where did that audit go that I was just failing"?